action.yml

```diff
@@ -82,25 +86,30 @@ runs:
           --output cov-analysis.tar.gz \
           --data "token=${TOKEN}&project=${{ steps.project.outputs.project }}"
       shell: bash
+      working-directory: ${{ inputs.working-directory }}
```

Comment:
i think you need to use `working-directory` above in the cache call.  otherwise that will be saving & restoring the `cov-analysis` dir in the cache in the default working directory, but this step will be downloading & unpacking it in the specified working directory.

Reply:
seems to pass.  ship it!